pkg/auth/role/error.go

```diff
@@ -0,0 +1,11 @@
+package role
```

Comment:
Maybe put this file directly into `pkg/auth/`? This way we avoid having a `auth/role` package with just an error definition and the call sites look sleeker: `auth.ErrNoValidRole`.

Reply:
✅